system/vmware-monitoring/templates/collector-configmap.yaml

```diff
@@ -146,9 +146,9 @@ data:
         key: "summary|accessible"
       - metric_suffix: "hostcount"
         key: "datastore|hostcount"
-      - metric_suffix: "cinder_tag"
+      - metric_suffix: "summary_tag"
         key: "summary|tagJson"
-      - metric_suffix: "cinder_state"
+      - metric_suffix: "summary_custom_tag_cinder_state"
         key: "summary|customTag:cinder_state|customTagValue"
 
     SDRSStatsCollector:
```

Comment:
Please deploy this change first to verify that the metrics are available as you expect.

Reply:
Thx Richard.

Sure I'll roll out the vmware-monitoring pipeline first.